docs/source/projects/architecture/content-architecture.rst

```diff
@@ -49,6 +49,7 @@ between the frontends and the IPython Kernel is described in
 The core execution machinery for the kernel is shared with terminal IPython:
 
 .. image:: figs/ipy_kernel_and_terminal.png
+   :alt: 
```

Comment:
```suggestion
   :alt: Venn diagram showing that Terminal IPython and the IPython Kernel both share the same Python executable. The diagram shows that Terminal IPython communicates with the Python executable using stdin and stdout, whereas the Python Kernel uses JSON messages sent over ZeroMQ. 
```

Reply:
Per Ely's suggestion, maybe we should set do blank alt-text: alt=""

Reply:
https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
Gabriel also mentioned in the sprint that perhaps ‘[aria-describedby](https://www.w3.org/TR/WCAG20-TECHS/ARIA15.html)’ tag could be used, which automatically make the alt-text point to an existing paragraph in the page, thus making full use of surrounding context. However, aria-tags are for HTML, and most formal documentation is in rst or markdown, which do not have aria labels.